redis/cluster.py

```diff
@@ -1450,29 +1419,9 @@ def initialize(self):
             # isn't a full coverage
             raise RedisClusterException(
                 f"All slots are not covered after query all startup_nodes. "
-                f"{len(self.slots_cache)} of {REDIS_CLUSTER_HASH_SLOTS} "
+                f"{len(tmp_slots)} of {REDIS_CLUSTER_HASH_SLOTS} "
```

Comment:
small bug fix

Reply:
Makes sense - limit based on available as opposed to the entire cluster. This should recover more cleanly.